controllers/Templates.php

```diff
@@ -52,6 +53,39 @@ private function getIndex()
         return ($final);
     }
 
+    function getXMLFromValidation($url, $paramsFromGet)
```

Comment:
This would be good to be in a service / library

Reply:
fixed, added to another individual class